contributing.md

```diff
@@ -70,3 +72,26 @@
 ### Reference
 * [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
 * [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+<br>
+
+## PR review rules
+ * Input from two reviewers is needed to merge a pull request. One reviewer approves the merge and the other reviewer merges the pull request.
+<br>
+
+## Issues
+### New issue
+Discovering an issue is great, here's what you need to do when you discover an issue;
+* Search if the issue has already been created.
+* If yes and open refer to existing issue.
+* If yes and closed reopen issue with descriptive comment.
+* If no, create the issue by;
+   * Following our issue [template.](https://github.com/pandas-dev/pandas/blob/master/.github/ISSUE_TEMPLATE.md)
+### Existing issue
+* Read comments.
+* Find out if anyone is working on it, if no, offer to do it. If yes, see if you can be of help.
+<br>
+
+## Reporting bugs
```

Comment:
Reporting bugs should be a subcategory of the issues, please move it under the Issues. And make its header `###`